### PR TITLE
feat: workspace catalog app ui steps 2.1.1

### PR DESCRIPTION
**Pull request by cwyl02**
modified from the project custom catalog app ui steps

## Checklist

- [ ] port to 2.2 once content is approved.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md [hunk 1]
```diff
@@ -23,7 +23,33 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
 export WORKSPACE_NAMESPACE=<workspace_namespace>
 ```
 
-## Deploy the application
+After creating a GitRepository, you can either use the Kommander UI or the CLI to deploy your custom applications.
+
+## Deploy the application using the Kommander UI
+
+Go to the Kommander UI to deploy your custom applications:
```
Comment (cbuto) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md:30]:
```suggestion
Go to the Kommander UI to deploy your catalog applications:
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md [hunk 2]
```diff
@@ -23,7 +23,33 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
...
+
+   ```yaml
+   someField: someValue
+   ```
+
+1. Once you confirm the details are correct, click the `Deploy` button.
+
+For all applications, you must provide a display name and an ID which is automatically generated based on what you enter for the display name, unless or until you edit the ID directly. The ID must be compliant with [Kubernetes DNS subdomain name validation rules](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names).
+
+Alternately, you can use the [CLI](#deploy-the-application-using-the-cli) to deploy your custom applications.
```
Comment (cbuto) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md:50]:
```suggestion
Alternately, you can use the [CLI](#deploy-the-application-using-the-cli) to deploy your catalog applications.
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md [hunk 3]
```diff
@@ -23,7 +23,33 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
...
+
+1.  Select the desired **Workspace**
+
+1.  Select **Applications** on the left navigation bar to browse the available applications from your configured repositories.
+
+1.  Select your desired application.
+
+1.  Select the version you'd like to deploy from the version drop-down, and then select Deploy. The `Deploy Workspace Catalog Application` page is displayed.
+
+1.  (Optional) If you want to override the default configuration values, copy your values content into the text editor under **Configure Service** or just upload your yaml file that contains the values:
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md:40]:
```suggestion
1.  (Optional) If you want to override the default configuration values, copy your customized values into the text editor under **Configure Service** or upload your yaml file that contains the values:
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md [hunk 4]
```diff
@@ -23,7 +23,33 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
...
+
+   ```yaml
+   someField: someValue
+   ```
+
+1. Once you confirm the details are correct, click the `Deploy` button.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md:46]:
```suggestion

    ```yaml
    someField: someValue
    ```

1.  Once you confirm the details are correct, click the `Deploy` button.
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md [hunk 5]
```diff
@@ -23,7 +23,33 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
 export WORKSPACE_NAMESPACE=<workspace_namespace>
 ```
 
-## Deploy the application
+After creating a GitRepository, you can either use the Kommander UI or the CLI to deploy your custom applications.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md:26]:
```suggestion
After creating a GitRepository, you can either use the Kommander UI or the CLI to deploy your catalog applications.
```

**Review by msdolbey — APPROVED** [review]
Some minor things - but other than that - looks good!
👍 
How about this for a custom robot

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md (hunk 1 above)
Comment (cwyl02) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md]: done

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md (hunk 2 above)
Comment (cwyl02) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md]: done

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md (hunk 4 above)
Comment (cwyl02) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md]: done

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md (hunk 5 above)
Comment (cwyl02) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md]: done

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md [hunk 6]
```diff
@@ -23,7 +23,33 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
 export WORKSPACE_NAMESPACE=<workspace_namespace>
 ```
 
-## Deploy the application
+After creating a GitRepository, you can either use the Kommander UI or the CLI to deploy your catalog applications.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md:26]:
```suggestion
After creating a GitRepository, use either the Kommander UI or the CLI to deploy your catalog applications.
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md [hunk 7]
```diff
@@ -23,7 +23,33 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
...
+
+## Deploy the application using the Kommander UI
+
+Go to the Kommander UI to deploy your catalog applications:
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md:30]:
```suggestion
Follow these steps to deploy your catalog applications from the Kommander UI:
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md [hunk 8]
```diff
@@ -23,7 +23,33 @@ Set the `WORKSPACE_NAMESPACE` environment variable to the name of the workspace'
...
+
+1.  Select the version you'd like to deploy from the version drop-down, and then select Deploy. The `Deploy Workspace Catalog Application` page is displayed.
+
+1.  (Optional) If you want to override the default configuration values, copy your customized values into the text editor under **Configure Service** or upload your yaml file that contains the values:
+
+    ```yaml
+    someField: someValue
+    ```
+
+1.  Once you confirm the details are correct, select the `Deploy` button.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/application-deployment/index.md:46]:
```suggestion
1.  Confirm the details are correct, and then select the `Deploy` button.
```

**Review by Colin1841 — APPROVED** [review]

**Pull request merged**